src/dotnet/APIView/APIViewWeb/Pages/Assemblies/RequestedReviews.cshtml

```diff
@@ -0,0 +1,142 @@
+﻿@page "RequestedReviews"
+@model APIViewWeb.Pages.Assemblies.RequestedReviews
+@using APIViewWeb.Helpers
+@using APIViewWeb.Models
+@{
+    ViewData["Title"] = "Requested Reviews";
+    var userPreference = PageModelHelpers.GetUserPreference(Model._preferenceCache, User.GetGitHubLogin()) ?? new UserPreferenceModel();
+    ViewData["UserPreference"] = userPreference;
+}
+<div class="mx-5 row">
+    <div class="col-md-12" id="reviews-filter-partial">
+        <div class="mt-3 row">
+            <div class="col-12 p-0 border rounded table-responsive shadow-sm">
+                <table id="reviews-table" class="table table-sm table-hover m-0" style="width:100%">
+                    <thead>
+                        <tr>
+                            <th scope="col" class="pl-4">Name</th>
+                            <th scope="col" class="border-left pl-3">Author</th>
+                            <th scope="col" class="border-left pl-3">Last Updated</th>
+                            <th scope="col" class="border-left pl-3">Type</th>
+                            <th scope="col" class="border-left pl-3">Approval Requested</th>
+                        </tr>
+                    </thead>
+                    <tbody>
```

Comment:
For the two tables on this page, let's introduce headers above each of them that read "Pending Reviews" and "Recently-Approved Reviews"